testsuite/unittests/test_binary.sli

```diff
@@ -66,14 +66,14 @@
 
```

Comment:
The test should also test that connections actually fail when one tries to connect binary neurons with other neurons.